release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml

```diff
@@ -0,0 +1,75 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
Shouldn't CRDs be defined in our build-tooling repo like here or are we moving them here instead? https://github.com/aws/eks-distro-build-tooling/tree/main/release/config/crds

Reply:
Having two repos is a constant source of extra work and the risk of more moving parts, which is why I put them here.

Reply:
Okay, I changed this to pull the CRDs out of the other repo